<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LotteryDraw</class>
 <widget class="QWidget" name="LotteryDraw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LotteryDraw</string>
  </property>
  <widget class="QLabel" name="pick_name">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>语方菲</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_pick">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>点名</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_stop">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>421</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_rand">
      <property name="text">
       <string>名单文件：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="filename_text"/>
    </item>
    <item>
     <widget class="QPushButton" name="select_file">
      <property name="text">
       <string>选择文件</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
